services/user/AdminSys/ui/src/pages/dashboard-page.tsx

```diff
@@ -1,6 +1,18 @@
-const DASHBOARDS_PREFIX =
-    window.location.hostname === "localhost"
-        ? "http://localhost:8091/grafana"
+function isLocalhost() {
+    const hostname = window.location.hostname;
+    const localhostPatterns = [
+        /^(.*\.)?localhost:.*$/,
```

Comment:
If `psinode` is running on port `80` or `443`, the hostname might not contain a port.